API Project/src/main/resources/config/application.properties

```diff
@@ -1,11 +0,0 @@
-# https://spring.io/guides/gs/accessing-data-mysql/
```

Comment:
This file is required for standalone mysql client access, why did you remove this?